### ISSUE TITLE
Enhancement Request: Add Customization Options for ChatBot Title and Image

### ISSUE THREAD

**Issue by damienLopa** [nvima/chat-ui#3]
**Issue Description:**
Currently, the title and image of the ChatBot are hardcoded within the codebase, limiting the ability to fully customize the appearance of the ChatBot component. It would greatly enhance the usability and flexibility of the ChatBot if we could pass custom props for the title and image, allowing developers to tailor these elements according to their application's design and branding.

**Proposed Solution:**
Introduce **title** and **img** props to the Chat component, enabling developers to easily customize the title and image displayed.

**Comment by fareed13** [comment]
Yes, I want to customize the chatbot header like the title and image. When a chatbot message response comes during this time, typing should appear.
